.github/workflows/variant_remapping.yml

```
@@ -0,0 +1,40 @@
+name: Variant Remapping
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        # Install Nextflow
+        mkdir /tmp/nextflow && cd /tmp/nextflow
+        wget -qO- get.nextflow.io | bash
+        echo "/tmp/nextflow" >> $GITHUB_PATH
+        cd -
+        # Install dependencies
+        # $CONDA is an environment variable pointing to the root of the miniconda directory
+        $CONDA/bin/conda env update --file conda.yml --name base
+        $CONDA/bin/conda run pip install -r requirements.txt
```

```suggestion
        $CONDA/bin/conda run pip -q install -r requirements.txt
```

This will still shows errors should they occur, but will reduce the log output overall

---

Done